pack-solution/action.yml

```diff
@@ -4,11 +4,11 @@ name: 'pack-solution'
 description: 'Power Platform Pack Solution'
 inputs:
   solution-file:
-    description: 'Path/filename of the local solution zip file'
+    description: 'Target Path/filename of the local solution file'
```

Comment:
nit: lower case 'path';
thanks for fixing the confusing copy/pasted help text, @gtrekter

Reply:
No worries. Thank you for developing the GH action ☺️

Reply:
I changed 'path' to lowercase